### DISCUSSION TITLE
hello when i doublle click at floorp browser nothing happen like it dosent oopen pls help

### DISCUSSION THREAD

**Discussion by RUOKexe** [Floorp-Projects/Floorp#687] [Q&A]
hello when i doublle click at floorp browser nothing happen like it dosent oopen pls help     it dosent open idk why pls help me try this browser

**Comment by creeper-0910** [comment]
Please tell us which operating system you are using.

**Reply by RUOKexe** [reply]
windows 10 22h2

**Comment by RUOKexe** [comment]
> Please tell us which operating system you are using.

**Comment by creeper-0910** [comment]
duplicate

**Reply by RUOKexe** [reply]
i am trying smth myb this will help fix it

**Comment by RUOKexe** [comment]
> duplicate
